### PR TITLE
[core] Fix buggy using alias

### PR DESCRIPTION
**Pull request by dentiny**
Please notice, exposing `using` alias in header is extremely dangerous and populate all dependent files.
See google coding style: https://google.github.io/styleguide/cppguide.html#Aliases

### CODE REVIEW

File: src/ray/raylet/worker_pool.cc [hunk 1]
```diff
@@ -39,10 +39,6 @@ DEFINE_stats(worker_register_time_ms,
 
 namespace {
 
-// Add this prefix because the worker setup token is just a counter which is easy to
-// duplicate with other ids.
-const std::string kWorkerSetupTokenPrefix = "worker_startup_token:";
```
Comment (dentiny) [inline src/ray/raylet/worker_pool.cc:44]: Unused anywhere.

**Comment by rynewang** [comment]
What does this new ssh_config file do?

**Comment by dentiny** [comment]
> What does this new ssh_config file do?

@rynewang sorry my bad, my devbox is broken today and I was re-setting up everything.

**Review by rynewang — APPROVED** [review]

**Comment by jjyao** [comment]
There are test failures

**Comment by dentiny** [comment]
> There are test failures

These are build failures, header include are contagious, have to rely on CI

**Comment by rynewang** [comment]
can you try to build this locally? I think you don't have to rely on the slow CI.

```
cd python
pip install -U -e ".[all]" -v
```

**Comment by dentiny** [comment]
> can you try to build this locally? I think you don't have to rely on the slow CI.
> 
> ```
> cd python
> pip install -U -e ".[all]" -v
> ```

Thank you!

**Pull request merged**